tests/v2/validation/pipeline/Jenkinsfile.multibranch.recurring

```diff
@@ -58,7 +59,7 @@ node {
                           string(credentialsId: 'RANCHER_VALID_TLS_KEY', variable: 'RANCHER_VALID_TLS_KEY'),
                           string(credentialsId: 'RANCHER_BYO_TLS_CERT', variable: 'RANCHER_BYO_TLS_CERT'),
                           string(credentialsId: 'RANCHER_BYO_TLS_KEY', variable: 'RANCHER_BYO_TLS_KEY'),
-                          string(credentialsId: 'PRIME_STAGING_REPO', variable: 'PRIME_STAGING_REPO'),
+                          string(credentialsId: 'PRIME_REPO', variable: 'PRIME_REPO'),
```

Comment:
qq, Why don’t we declare PRIME_REPO as def PRIME_REPO = "" directly in the code since it’s only a public URL, rather than using string(credentialsId: 'PRIME_REPO', variable: 'PRIME_REPO')

Reply:
Discussed this offline